packages/palette_generator/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 0.3.4
 
 * Updates minimum supported SDK version to Flutter 3.3/Dart 2.18.
 * Aligns Dart and Flutter SDK constraints.
+* Add option to run `PaletteGenerator.from` functions in an isolate
```

Comment:
Please see the style guide linked from the PR checklist; this needs to use the right verb form and punctuation as described there.

Reply:
Fixed in 7b30aaaff.